test/http.facility.test.js

```diff
@@ -348,6 +348,32 @@ describe('http facility tests', () => {
 
       expect(fs.existsSync(writefile)).to.be.true()
     })
+
+    it('should support qs params', async () => {
+      app.get('/foo/bar/9', (req, res) => {
+        res.json(req.query)
+      })
+
+      let resp = await fac.request('/foo/bar/9', { method: 'get', encoding: 'json', qs: { a: 1, b: 'c' } })
```

Comment:
could we also check the cases when fac has default `qs`, please?

Reply:
yep we already have it, see line 372